autoload/codefmt/formatterhelpers.vim

```diff
@@ -0,0 +1,59 @@
+" Copyright 2017 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+
+""
+" @public
+" Format lines in the current buffer via a formatter invoked by {cmd}. The
```

Comment:
Perhaps document that `{cmd}` is something you can pass to `maktaba#syscall#Create()`? As it stands, I wasn't sure if this was requiring a Syscall object, and the cases above where you passed a list were bugs.

(also below.)

Reply:
Oh, good catch. It was a bug (but strangely wasn't caught by the tests… I'll need to investigate coverage there).

It actually does take a maktaba.Syscall… updated to clarify that and enforce.

Reply:
I mean, I think there's some advantage in being able to pass in anything that can be passed to syscall#Create(), since then we can pass a list in the simple case, and a Syscall object in the uncommon case?

Reply:
Oh yeah! Good idea, I forgot we made it that clever. And I guess it didn't fail tests because it wasn't broken. :man_facepalming:

Updated to accept both.